content/sensu-go/5.7/guides/troubleshooting.md

```diff
@@ -64,11 +69,28 @@ following those logs are described. The name of the desired service, e.g.
 | Ubuntu       | <= 14.10   | log file     | {{< highlight shell >}}tail --follow /var/log/sensu/sensu-${service}{{< /highlight >}} |
 | Debian       | >= 8       | journald     | {{< highlight shell >}}journalctl --unit sensu-${service} --follow{{< /highlight >}}   |
 | Debian       | <= 7       | log file     | {{< highlight shell >}}tail --follow /var/log/sensu/sensu-${service}{{< /highlight >}} |
-| Windows      | Any        | log file     | {{< highlight powershell >}}Get-Content -  Path "C:\scripts\test.txt" -Wait{{< /highlight >}}
 
 _NOTE: Platform versions described above are for reference only and do not
 supercede the documented [supported platforms][platforms]._
 
+{{< platformBlockClose >}}
+
+{{< platformBlock "Windows" >}}
+
+#### Windows
+
+The Sensu agent stores service logs to the location specified by the `log-file` configuration flag (default: `C:\\ProgramData\\sensu\\log\\sensu-agent.log`).
```

Comment:
The default is `%ALLUSERSPROFILE%\Sensu\log\sensu-agent.log` which on a standard installation of Windows will default to what's currently listed (minus the double-backslashes).

The double-backslashes are only shown in the terminal to prevent the backslashes from being treated as escape characters and probably aren't necessary for the docs.